tensorflow/python/lib/io/file_io.py

```diff
@@ -586,6 +589,26 @@ def is_directory_v2(path):
   except errors.OpError:
     return False
 
+def has_atomic_move(path):
+  """ Returns whether or not the file system of the given path
+      supports the atomic move operation for file or folder.
+      If atomic move is supported, it is recommended to use a temp location
+      for writing and then move to the final location. 
+  Args:
+    path: string, path to a file
+    
+  Returns:
+    True, if the path is on a file system that supports atomic move
+    False, if the file system does not support atomic move. In such cases
+           we need to be careful about using moves. In some cases it is safer
+           not to use temporary locations in this case.
+  """
+  try:
+    return _pywrap_file_io.HasAtomicMove(compat.as_bytes(path))
+  except errors.OpError:
+    # defaults to True
+    return True
```

Comment:
Strange that the test is failing internally when it passes on my end, it might have to do with this except block here, and some actual error is being suppressed. Let me remove this try catch block.

Reply:
It might help to see what's the error thrown if indeed this is the reason.